### DISCUSSION TITLE
umi3.5 mfsu方案构建，访问浏览器域名 速度大幅下降

### DISCUSSION THREAD

**Discussion by LukeLiou** [umijs/umi#10479] [Q&A]
请问，下面这个问题有解决方案吗？
使用umi3.5 mfsu方案构建流水线后，访问域名，发现http请求数剧增，网站访问速度大幅下降
目测和config.js中配置有关：`mfsu: { production: { output: '.mfsu-production' } },`

**Comment by zhuchongyue** [comment]
遇到你相同的问题，一堆mf-vaXXX 和 mf-depXXX的请求。

有人解决这个问题么

**Reply by LukeLiou** [reply]
目前我发现的解决方案就是，在config.js配置中，只保留 webpack5: {}，不开启 mfsu，这样也可以达到流水线构建加速的效果
